graphql/tests/introspection.rs

```diff
@@ -449,7 +449,7 @@ fn expected_mock_schema_introspection() -> q::Value {
             q::Value::List(vec![object_value(vec![
                 ("name", q::Value::String("language".to_string())),
                 ("description", q::Value::Null),
-                ("defaultValue", q::Value::String("English".to_string())),
+                ("defaultValue", q::Value::String("\"English\"".to_string())),
```

Comment:
How comes this got the extra quotes? Is this desirable?

Reply:
Yep! E.g. in GraphiQL, you'd want
```graphql
someField(
  someInput = "default string"
)
```
to be shown instead of
```graphql
someField (
  someInput = default string
)
```

Reply:
ah interesting